css/features.css

```diff
@@ -0,0 +1,45 @@
+.features--pd-btm-0 {
+  padding-bottom: 0;
+}
+
+.features__list {
+  display: flex;
+  gap: 30px;
+
+  list-style: none;
+}
+
+.features__item {
+  width: 270px;
+}
+
+.features__icon-background {
+  height: 120px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 4px;
+
+  background-color: var(--section-color);
```

Comment:
Variable names should be meaningful. Better name for this variable `--color-grey`